scripts/deblur

```diff
@@ -34,6 +34,9 @@ from deblur.workflow import (launch_workflow, trim_seqs, dereplicate_seqs,
 
 __version__ = "0.9.2.1"
 
+default_error_dist = '1,0.06,0.02,0.02,0.01,0.005,0.005' \
```

Comment:
We want to have a default error profile (from our experience with illumina and mock mixture analysis). However, we want to allow the user 2 alternative ways to supply his own error profile:
1. write it as a comma separated list for each hamming distance
2. supply a maximal hamming 1 read error (read_error) and create the hamming 2,3 etc. profile from it (less good since the assumption you can predict hamming 2 error prob. from hamming 1 is incorrrect).
* But the user cannot supply both together.

Since if we supply the default error profile as a default value to error_dist, we won't be able to sense if the user supplied both error_dist and read_error and warn about it (which seems to me some users will make this mistake), we instead have the default error profile as default_error_dist, and if error_dist is None, we do error_dist = default_error_dist

But other ideas will be welcome :)